storage/Filesystems/BtrfsImpl.cc

```diff
@@ -59,14 +60,16 @@ namespace storage
 
     Btrfs::Impl::Impl()
 	: BlkFilesystem::Impl(), configure_snapper(false), snapper_config(nullptr),
-	  metadata_raid_level(BtrfsRaidLevel::DEFAULT), data_raid_level(BtrfsRaidLevel::DEFAULT)
+	  metadata_raid_level(BtrfsRaidLevel::DEFAULT), data_raid_level(BtrfsRaidLevel::DEFAULT),
```

Comment:
NP: Since this initializer list now spans more than one line anyway, it would make sense to put each one on a separate line so the second one doesn't get overlooked that easily.

Same for the other constructor.

Reply:
OTOH you can save the effort of just using the default constructor; this is what happens anyway for non-primitive data types (anything else than char, int, bool) when you don't explicitly write it down.

Reply:
Yes, I know. I am simply doing the same as the rest of constructors, see for example `BlkFilesystem::Impl::Impl()`. We are always explicitly calling the constructor for each attribute, so I think we should continue doing the same (to not mix styles).